src/xunit.netcore.extensions/Interop.PlatformDetection.cs

```diff
@@ -11,14 +11,15 @@ internal static partial class Interop
 {
     internal enum OperatingSystem
     {
-        Windows,
+        FreeBSD,
         Linux,
-        OSX
+        OSX,
+        Windows
     }
 
-    internal static bool IsWindows
+    internal static bool IsFreeBSD
     {
-        get { return OperatingSystem.Windows == PlatformDetection.OperatingSystem; }
+        get { return OperatingSystem.FreeBSD == PlatformDetection.OperatingSystem; }
```

Comment:
I have added the code in `PlatformDetection.OperatingSystem` and amended the commit.